consoles/sshXtermVt.pm

```diff
@@ -61,4 +61,10 @@ sub kill_ssh {
     $self->backend->stop_ssh_serial;
 }
 
+# when use ipmi backend with ssh , reboot code should not be invoked in ssh
```

Comment:
I don't understand this comment. So when *should* this be used? Why should this console also be special in this regard? I suggest to extend the comments for the methods in this file with proper POD format to describe what in details is necessary and special. How does this method relate to the above "kill_ssh"?

Reply:
> I don't understand this comment. So when _should_ this be used? 

in the context of installation on remote ssh connection .
1. backend monitor the socket rate .
2. ssh connection killed may cause half-open
3. ssh connection will be killed after install 

here , basically no stander way close ssh connectison . because last step of installation is reboot , which kill ssh

4. kill-ssh  method will remove the sock from monitor .



  
> Why should this console also be special in this regard?

Well , I'm not sure , i hope you can tell where to add.
 but this issue is about ssh connection can not disconnect normally ,
 only this type console have this issue. maybe should add to all ssh relate console. 

>I suggest to extend the comments for the methods in this file with proper POD format to describe what in details is necessary and special. How does this method relate to the above "kill_ssh"?

refer to 4 in above

Reply:
update new comment